scrubber/utils.py

```diff
@@ -8,11 +8,12 @@
 
 import numpy as np
 
-def optimize_conformers(mol: Mol, use_mmff: bool=True, max_ff_iter: int = 400):
+def optimize_conformers(mol: Mol, ff: str="mmff94", max_ff_iter: int = 400):
```

Comment:
I think we should raise an error if `ff == espaloma`. The if/else block below was written when the only options were MMFF and UFF. But now it's in an inconsistent state because `--ff uff` changes what happens with `--ring_minimize` as per the user request, but `--ff espaloma` silently defaults to UFF. There could be also an error at the command line argument parsing level, stating that `--ff espaloma` and `--ring_minimize` are incompatible.